src/docs/release/breaking-changes/use-maxLengthEnforcement-instead-of-maxLengthEnforced.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Use maxLengthEnforcement instead of maxLengthEnforced
+description: Introducing the MaxLengthEnforcement enum.
+---
+
+## Summary
+
+_If you don't use the `maxLength` or `maxLengthEnforced` properties in a
+`TextField`, `TextFormField`, or `CupertinoTextField`,
+or you don't care about the truncation behavior of an
+editable text field, you can stop reading._
+
+To control the behavior of `maxLength`
+in the `LengthLimitingTextInputFormatter`,
+use `maxLengthEnforcement` instead of `maxLengthEnforced`, which was deprecated.
+
+## Context
+
+The `maxLengthEnforced` parameter was used to decide whether text fields should
+truncate the input value when it reaches the `maxLength` limit. Additionally,
+`TextField` and `TextFormField` will display a warning message in the
+characters counter.
```

Comment:
... when this parameter is set to false and the length of the user input exceeds `maxLength`.